qa/automation/CareerPortal.Tests.Scenarios/samsonova_sendCV/1.1.send_cv.feature

```diff
@@ -0,0 +1,40 @@
+  Feature: 1.1.Send CV
+
+  Scenario: 1.1_01 positive sending CV
+  Given user opened the form "Send CV"
+  When user enters valid data in the field "Name"
+  And user enters valid data in the field "Email"
+  And user attaches CV
+  And user marks "I agree to the personal data processing"
+  And user clicks the "Send" button
+  Then my CV has been send and success message should appear
+
+
+  Scenario: 1.1_02 exceeding the limit of characters in the field "Name"
+  Given user opened the form "Send CV"
+  When user enters 71 characters with space delimiter in the field "Name"
```

Comment:
I guess you can add some tests, for example with happy path (positive), because according to this task we need to use different test design techniques and it will be great to use their in that case.

Reply:
thanks, no, I want to make ready tests better